lib/common.ts

```diff
@@ -0,0 +1,48 @@
+import { BalenaSDK } from 'balena-sdk';
+import { ExpectedError } from './errors';
+
+// TODO fix this any
+export function getReleaseTagKeysAndValues(options: any) {
```

Comment:
This function only uses `options` to access `options['release-tag']`. So what about changing its signature to be:
```
export function parseReleaseTagKeysAndValues(releaseTags: string[]) {
```
So the caller does:
```
const { releaseTagKeys, releaseTagValues } = parseReleaseTagKeysAndValues(
    options['release-tag'] ?? [],
);
```
This way, you don't need to worry about typing `options` in a way that covers both the `push` and `deploy` commands. I also renaming it to `parse` instead of `get`.